projects/sport-data-valley/seeds/questionnaires/daily_questionnaire_rheumatism.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+db_title = ''
+db_name1 = 'Dagelijkse vragenlijst reuma'
+dagboek1 = Questionnaire.find_by(key: File.basename(__FILE__)[0...-3])
+dagboek1 ||= Questionnaire.new(key: File.basename(__FILE__)[0...-3])
+dagboek1.name = db_name1
+
+class RheumatismMethods
+  class << self
+    def category_title(category, title)
+      "<strong>#{category}</strong>: #{title}"
+    end
+  end
+end
+
+dagboek_content = [
+  {
+    id: :v1,
+    type: :range,
+    title: RheumatismMethods.category_title('Vermoeidheid', 'Hoe hevig was uw vermoeidheid gemiddeld vandaag?'),
+    min: 0,
+    max: 10,
+    step: 0.1,
+    required: true,
+    no_initial_thumb: true,
+    labels: ['geen vermoeidheid', 'meest voorstelbare vermoeidheid']
+  }, {
+    id: :v2,
+    type: :radio,
+    title: RheumatismMethods.category_title('Vermoeidheid', 'Was er vandaag sprake van een ernstige piek in vermoeidheid?'),
+    options: ['Nee'],
```

Comment:
shouldn't Ja also be an option? or how does that work?

Reply:
So "Ja" is the otherwise option here. Because when you select "Ja", you have to specify a reason in a text input. And they didn't describe that as being a separate question, just as a way to answer this question. The only way to do that in the questionnaire engine currently is by using the "otherwise" option, and then overwriting the default otherwise text with whatever you specify.

It looks like this:
![chrome_2021-12-15_21-14-44](https://user-images.githubusercontent.com/607965/146258504-b8d707d3-8372-4fea-8c01-b3cfec2ec00d.png)

Reply:
And if you store it, it looks like this

```ruby
{"locale"=>"nl",
 "v1"=>"5.8",
 "v2"=>"Ja<br />Zo ja, waar denkt u dat dit door kwam?",
 "v2_ja_br_zo_ja_waar_denkt_u_dat_dit_door_kwam_text"=>"geen idee",
 "v3"=>"2.8",
 "v4"=>"7.4",
 "v5"=>"4",
 "v6"=>"2",
 "v7"=>"5.5",
 "v8"=>"7",
 "v1_timing"=>"37793",
 "v2_nee_timing"=>"1648",
 "v3_timing"=>"1132",
 "v4_timing"=>"1046",
 "v5_timing"=>"1558",
 "v6_timing"=>"1811",
 "v7_timing"=>"1257",
 "v8_timing"=>"3654",
 "v2_ja_br_zo_ja_waar_denkt_u_dat_dit_door_kwam_timing"=>"82",
 "v2_ja_br_zo_ja_waar_denkt_u_dat_dit_door_kwam_text_timing"=>"3961"}
```

Reply:
You can just check if the answer for v2 starts with "Ja"